code/extensions/CommentsExtension.php

```diff
@@ -581,6 +581,8 @@ protected function updateModerationFields(FieldList $fields)
                     $spamGrid
                 )
             );
+            $tabs->setTitle(_t('CommentsExtension.COMMENTSTABSET', 'Comments'));
```

Comment:
Maybe this could be `CommentsExtension. COMMENTOPTIONS` which is already defined at "Comments"? Otherwise could you please add this to `lang/en.yml` as well?